experimental/dds/tree2/.dependency-cruiser.js

```diff
@@ -0,0 +1,436 @@
+/** @type {import('dependency-cruiser').IConfiguration} */
+module.exports = {
+	forbidden: [
+		{
+			name: "no-circular",
+			severity: "error",
+			comment:
+				"This dependency is part of a circular relationship. You might want to revise " +
+				"your solution (i.e. use dependency inversion, make sure the modules have a single responsibility) ",
+			from: {},
+			to: {
+				circular: true,
+			},
+		},
+		{
+			name: "no-orphans",
+			comment:
+				"This is an orphan module - it's likely not used (anymore?). Either use it or " +
+				"remove it. If it's logical this module is an orphan (i.e. it's a config file), " +
+				"add an exception for it in your dependency-cruiser configuration. By default " +
+				"this rule does not scrutinize dot-files (e.g. .eslintrc.js), TypeScript declaration " +
+				"files (.d.ts), tsconfig.json and some of the babel and webpack configs.",
+			severity: "error",
+			from: {
+				orphan: true,
+				pathNot: [
+					"(^|/).[^/]+.(js|cjs|mjs|ts|json)$", // dot files
+					".d.ts$", // TypeScript declaration files
+					"(^|/)tsconfig.json$", // TypeScript config
+					"(^|/)(babel|webpack).config.(js|cjs|mjs|ts|json)$", // other configs
+				],
+			},
+			to: {},
+		},
+		{
+			name: "no-deprecated-core",
```

Comment:
I think it might make sense to strip out most of the content of this file for now, so we can start simple.

Some of these rules are better covered by our existing linter configuration, or just not very useful. For example, our non-test code shouldn't depend on node at all, so banning this subset of deprecated node APIs isn't very valuable.

Reply:
but is there a rule in place to avoid the test code depending on deprecated node APIs? This rule can be usefull for that popouse.